### PR TITLE
java/example: Align JDBC javadoc comment with the other example's com…

### PR DESCRIPTION
**Pull request by michael-berlin**
…ment.

### CODE REVIEW

**Comment by bbeaudreault** [comment]
LGTM

**Comment by thompsonja** [comment]
LGTM

File: java/example/src/main/java/io/vitess/example/VitessJDBCExample.java [hunk 1]
```diff
@@ -11,7 +11,10 @@
 import org.joda.time.Instant;
 
 /**
- * Created by harshit.gangal on 10/03/16.
+ * VitessClientExample.java is a sample for using the Vitess JDBC driver.
```
Comment (ashudeep-sharma) [inline java/example/src/main/java/io/vitess/example/VitessJDBCExample.java]: VitessClientExample or VitessJDBCExample?

Comment (michael-berlin) [inline java/example/src/main/java/io/vitess/example/VitessJDBCExample.java]: good catch :) I've fixed it.

**Pull request merged**